solutions/validation/validate-pre-install.py

```diff
@@ -39,27 +56,37 @@
 ose_package_not_installed_dict = {}
 ssh_connection = HandleSSHConnections()
 selinux_dict = {}
-ose_repos = ["rhel-7-server-rpms", "rhel-7-server-extras-rpms", "rhel-7-server-ose-3.1-rpms"]
+if options.openshift_version:
+    if "3.2" in options.openshift_version:
+        openshift_server_repo = "rhel-7-server-ose-3.2-rpms"
+    elif "3.1" in options.openshift_version:
+        openshift_server_repo = "rhel-7-server-ose-3.1-rpms"
+    elif "3.3" in options.openshift_version:
```

Comment:
I think a regex validation of the option would be the ideal case... I would also (for now) restrict it to a format of "3.x"